frontend/mock-db/init.ts

```diff
@@ -4,14 +4,27 @@ import * as reblogAPI from 'wildebeest/functions/api/v1/statuses/[id]/reblog'
 import { statuses } from 'wildebeest/frontend/src/dummyData'
 import type { Account, MastodonStatus } from 'wildebeest/frontend/src/types'
 
+/* eslint-disable @typescript-eslint/no-empty-function */
 const kek = 'test-kek'
 const queue = {
 	async send() {},
 	async sendBatch() {},
 }
-const kv_cache: any = {
+const kv_cache: KVNamespace = {
```

Comment:
Can we leave the any here? The other KV methods will never be called, it's dead code.

Reply:
yeah... I did feel like it was a bit too much...

ok I can revert it back 😢

Reply:
actually, I removed the `: any` and added a cast when we're using the variable, I hope that's fine 🙂